configs/twilio.json

```diff
@@ -1,7 +1,13 @@
 {
   "index_name": "twilio",
   "start_urls": [
-    "https://www.twilio.com/docs"
+    {
+      "url": "https://www.twilio.com/docs/studio",
+      "tags": ["studio"]
+    },
+    {
+      "url": "https://www.twilio.com/docs"
+    },
```

Comment:
Please remove this last coma to make it a valide JSON. You do not need to wrap this URL instead an object btw